pyrene/src/components/DateTimePicker/DateTimeInput/DateTimeInput.tsx

```diff
@@ -0,0 +1,139 @@
+import React, { forwardRef } from 'react';
+import clsx from 'clsx';
+
+import { allowedValueCheck } from '../../../utils/DateUtils';
+
+import Icon from '../../Icon/Icon';
+import styles from './dateTimeInput.css';
+
+
+export interface DateTimeInputProps {
+  /**
+   * Date format used by component
+   */
+  dateFormat?: string,
+  /**
+   * Boolean to toggle time display
+  */
+  dateOnly?: boolean,
+  /**
+   * Input date value
+  */
+  dateValue?: string,
+  /**
+   * Input error
+  */
+  errorValue?: string,
+  /**
+   * Component is disabled
+  */
+  disabled?: boolean,
+  /**
+   * Input field label
+  */
+  label?: string,
+  /**
+   * Input field name
+  */
+  name?: string,
+  /**
+   * Onchange from parent. Currently this handles onchange function passed from react-datepicker
+  */
+  onChange?: (event: React.ChangeEvent<HTMLInputElement>) => void,
+  /**
+   * Onclick from parent. Currently this handles onclick function passed from react-datepicker
+  */
+  onClick?: () => void,
+  /**
+   * Onfocus from parent.
+  */
+  onFocus?: () => void,
+  /**
+   * Callback to set Date value in parent
+   */
+  setDateValue?: (value: string) => void,
+  /**
+   * Callback to set Time value in parent
+   */
+  setTimeValue?: (value: string) => void,
+  /**
+   * Time format used by component
+   */
+  timeFormat?: string,
+  /**
+   * Input time value
+  */
+  timeValue?: string,
+}
+
+const DateTimeInput = forwardRef(({
+  dateFormat = 'dd.MM.yyyy',
+  dateOnly = false,
+  dateValue = '',
+  errorValue,
+  disabled,
+  label = dateOnly ? 'Date' : 'Date & Time',
+  name,
+  onChange,
+  onClick,
+  onFocus,
+  setDateValue,
+  setTimeValue,
+  timeFormat = ' HH:mm',
```

Comment:
Because technically you can pass in a format like `dd.MM.yyyyHH:mm` and it will work. so if you need spacing between the date and time it should be included in the format.